.erb-lint.yml

```diff
@@ -38,7 +38,7 @@ linters:
           - 'range-light'
           - 'field-dark'
           - 'progress'
-          - 'flex-(column|none)'
+          - 'flex(-(center|baseline|stretch|start|end|grow|none|first|last))?'
```

Comment:
Note: Not all BassCSS classes are included here, and in the case of `flex-column`, I removed it from being disallowed. This is because these classes also exist as design system utilities. In most cases, the specific implementation is similar/identical, and the design system will take precedent.